presentation/backend/src/presentation-backend/SelectionScopesHelper.ts

```diff
@@ -272,3 +239,17 @@ const skipTransients = (callback: (id: Id64String) => void) => {
       callback(id);
   };
 };
+
+interface ComputeElementSelectionScopeProps {
+  iModel: IModelDb;
+  elementIds: Id64String[];
+  level: number;
+}
+function createElementSelectionScopeProps(requestOptions: SelectionScopeRequestOptions<IModelDb>, ids: Id64String[], scopeParams?: SelectionScopeParams) {
+  const level = (scopeParams && typeof scopeParams === "object" && typeof scopeParams.level === "number") ? scopeParams.level : 0;

```

Comment:
It seems falsy `scopeParams` check could be omitted because its type is being checked.

Reply:
This was actually required, because `scopeParams` were being passed straight from `PresentationRpcImpl` which received them as `null` rather than `undefined`. I was going to fix this, but it's now irrelevant after I added a separate overload for `computeSelection`.